blockprotocol/blockprotocol
# Add "Payment History" section to the billing page

## 🌟 What is the purpose of this PR?




This PR adds a "Payment History" section to the billing page.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203623179643287/1203781148500078/f) _(internal)_

## 🔍 What does this change?




- adds a `PaymentHistory` component

## 🛡 What tests cover this?



Manual testing.

## ❓ How to test this?



1. Checkout the branch and setup the local dev environment
2. Run the internal api locally on the `bw/payment-history` branch (if it hasn't merged yet)
3. Login and subscribe to a subscription tier
4. Observe the billing history section on the billing page

## 📹 Demo



<img width="957" alt="image" src="https://user-images.githubusercontent.com/42802102/221391569-f367b5b1-46f3-46ff-b76c-c4ae211b7cd8.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204060610003539
